### ISSUE TITLE
Traffic Ops: column s.ip6_address_is_service does not exist

### ISSUE THREAD

**Issue by zrhoffman** [apache/trafficcontrol#4514]
## I'm submitting a ...

- [x] bug report
- [ ] new feature / enhancement request
- [ ] improvement request (usability, performance, tech debt, etc.)
- [ ] other 

## Traffic Control components affected ...

- [x] CDN in a Box
- [ ] Documentation
- [ ] Grove
- [ ] Traffic Control Client
- [ ] Traffic Monitor
- [ ] Traffic Ops
- [ ] Traffic Ops ORT
- [ ] Traffic Portal
- [ ] Traffic Router
- [ ] Traffic Stats
- [ ] Traffic Vault
- [ ] unknown

## Current behavior:

Receiving this error when requesting `GET /api/2.0/servers`:

```
ERROR: api.go:181: 2020-03-17T19:48:21.967972479Z: [fc01:9400:1000:8::4]:47686 querying: pq: column s.ip6_address_is_service does not exist
```

## Expected / new behavior:

No error when requesting `GET /api/2.0/servers`

## Minimal reproduction of the problem with instructions:

Build CDN-in-a-Box trafficops container at commit eb9cf79ea8 (#4142) or later

## Anything else:

The exact line in Traffic Ops executing the query that results in the error is [server/servers.go:274](https://github.com/apache/trafficcontrol/blob/master/traffic_ops/traffic_ops_golang/server/servers.go#L274).

**Comment by mhoppa** [comment]
did you rebuild the trafficops_perl container as well - I believe that is the container that puts the schema into place

**Comment by zrhoffman** [comment]
I thought so, but let me try again just to make sure.

**Comment by zrhoffman** [comment]
That was the issue. My bad, closing this.
